primitives/src/lib.rs

```diff
@@ -30,6 +30,9 @@ extern crate std;
 #[macro_use]
 extern crate serde;
 
+#[cfg(feature = "hex")]
+pub extern crate hex_stable as hex;
```

Comment:
Is there a technical reason to use the `extern crate` syntax over `pub use hex_stable as hex;`?